lib/routes/admin-api/tag.test.js

```diff
@@ -77,39 +77,20 @@ test('should be able to get single tag by type and value', t => {
         });
 });
 
-test('trying to get non-existing tag should not be found', t => {
```

Comment:
why is this test not relevant anymore?

Reply:
Because we don't use ids for tags anymore :)

Reply:
So it's basically testing the exact same thing the test on L91 is testing just with `(type=id, value=1125)` as parameters

Reply:
right, makes sense =)